modules/core/include/opencv2/core/private.hpp

```diff
@@ -317,6 +317,14 @@ static inline IppDataType ippiGetDataType(int depth)
         (IppDataType)-1;
 }
 
+#if defined(CV_COMPATIBLE_API)
+static inline IppDataType ippiGetDataType(int depth)
```

Comment:
It is "private" header. Avoid compatibility stuff here.

Reply:
It is utilized by the imgproc module though.. Is that really alright?